### DISCUSSION TITLE
Phantom pull requests issue with Dependabot

### DISCUSSION THREAD

**Discussion by aaroncawte** [community/community#37530] [Code Security]
Hi, I'm experiencing an issue with Dependabot version updates on a large monorepo regarding the open pull request limit. Set to the default of 5 for npm dependencies, Dependabot had stopped opening pull requests after a long period of successful operation claiming that too many were already open. However, when searching through our open PRs, we found that not to be the case: none of Dependabot's PRs were still open. 

To diagnose, we increased our limit to 10, and Dependabot promptly created 5 new pull requests, affirming our belief that 5 "phantom" pull requests are stopping Dependabot from behaving normally. 

Is there any way we can see which PRs Dependabot is stuck on, clear any cached data, or otherwise understand what's going wrong? Thanks

**Comment by ossanna16** [comment]
Hi there @aaroncawte and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
